engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsAtomNode.java

```diff
@@ -32,7 +34,7 @@ public static EqualsAtomNode build() {
     return EqualsAtomNodeGen.create();
   }
 
-  public abstract boolean execute(Atom left, Atom right);
+  public abstract boolean execute(VirtualFrame frame, Atom left, Atom right);
```

Comment:
Alas, @Akirathan, I had to add `frame` argument to all `Equals*Node`s. The possible conversion require context of `VirtualFrame`. As such the diff is pretty huge...